### DISCUSSION TITLE
Existing Database

### DISCUSSION THREAD

**Discussion by tataDan** [serverpod/serverpod#1313] [Q&A] [answered]
Is it possible to use serverpod to access an existing Postgres database?

**Comment by starklord** [comment, marked as answer]
Yes, it is possible configuring the folder "config" (where is the development.yaml and others) correctly, also creating the necesary serverpod tables inside the public scheme.
BTW i tried to use an existing postgresql database with no success because serverpod doesnt support all the scenarios(in my case my databases uses schemas and by now serverpod doesnt support schemas)
